_config.yml

```diff
@@ -973,3 +973,10 @@ podcast:
       file_byte_length: "33427019"
       duration: "00:34:53"
       credits: "Jon Allured, Ash Furrow, & Steve Hicks"
+    - title: "5. The Structure of Artsy's Product Team"
+      date: "2021-02-18 07:07:07 -0600"
+      description: "Jon Allured and Joanna Brocksen discuss the structure of Arty's Product team. What even is a PDDE??"
+      url: "https://artsy-engineering-podcast.s3.amazonaws.com/ArtsyEngineeringRadio-ArtsyProductTeams.mp3"
+      file_byte_length: "22241410"
+      duration: "00:28:50"
+      credits: "Jon Allured & Joanna Brocksen"
```

Comment:
non-blocking: Are you sure you don't want any `links`? While editing I remember hearing you mention a few links to share in the notes -- I even wrote them down (and forgot about them until right now): 

- ash's blog post about knowledge share
- something about pdde org?
- artsy.net/jobs
- @artsyopensource on twitter
- @jonallured on twitter

Reply:
Yeah I guess you're right - I added some with 00d6d16, how does that look?

Reply:
:chefskiss: